source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,161 @@
+.. _rm-convert-queries:
+
+===============
+Convert Queries
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can convert SQL queries to MongoDB syntax with query converter. 
+Query converter considers the mapping rules and schema 
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      a. If it is your first time using the query converter in your 
+         project, click :guilabel:`Paste SQL Query`.
+         If your project already has converted SQL code, click the 
```

Comment:
```suggestion
         If you have previously converted SQL code in your project, click the
```

My goal with this suggestion is to resolve the ambiguity between whether "converted" is an adjective or a verb.